domains/agung.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "agung",
```

username doesnt match github username
```suggestion
        "username": "agungsantoso",
```